tests/backstop_data/engine_scripts/puppet/mosaico/main-page.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = async (engine, scenario, viewport) => {
+  await engine.waitFor('.crm-mosaico-page');
+  // wait for images to load
+  await engine.evaluate(async () => {
```

Comment:
why isn't this done via `waitFor()`?

Reply:
WaitFor only make sure that `<img />` tag is visible on the screen but don't ensure that it is fully rendered on the screen.